pkg/report/template.go

```diff
@@ -102,27 +112,75 @@ func renderTemplates(dibReport Report) error {
 		return err
 	}
 
-	// Generate test.html
-	if err := dibReport.renderTemplate("test", nil); err != nil {
-		return err
+	return nil
+}
+
+// printReportURL return a string representing the path from which we can browse HTML report.
+func printReportURL(dibReport Report) string {
+	// GitLab context
+	gitlabJobURL := os.Getenv("CI_JOB_URL")
+	if gitlabJobURL != "" {
+		return fmt.Sprintf("%s/artifacts/file/%s/index.html", gitlabJobURL, dibReport.GetRootDir())
 	}
 
-	return nil
+	// Local context
+	finalReportURL, err := filepath.Abs(dibReport.GetRootDir())
+	if err != nil {
+		return dibReport.GetRootDir()
+	}
+
+	return fmt.Sprintf("%s/index.html", finalReportURL)
```

Comment:
Can we have an absolute path with `file://` prefix so we can `ctrl` + `click` from the terminal and directly open it ?

```suggestion
	return fmt.Sprintf("file://%s/index.html", finalReportURL)
```